packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -332,7 +309,7 @@ class WebSettings {
   ///
   /// The default is false.
   Future<void> setSupportMultipleWindows(bool support) {
-    return api.setSupportZoomFromInstance(this, support);
+    return api.setSupportMultipleWindowsFromInstance(this, support);
```

Comment:
This seems like a change that should have an associated test.

Reply:
I added a test for this and the rest of the methods in the `android_webview.dart` file.